npm-shrinkwrap.json

```diff
@@ -4682,7 +4682,7 @@
         "form-data": {
           "version": "0.2.0",
           "from": "form-data@>=0.2.0 <0.3.0",
-          "resolved": "http://registry.npmjs.org/form-data/-/form-data-0.2.0.tgz",
+          "resolved": "https://registry.npmjs.org/form-data/-/form-data-0.2.0.tgz",
```

Comment:
Weird how a bunch of these have spontaneously switched to `https:` ?

Reply:
...and that change is now causing errors on Travis :sob:

Reply:
Yeah, but they should've never been `http:` in the first place as far as I can see.

Reply:
Lets omit the `https` -> `http` changes from this PR to make Travis happy and get this merged ASAP